index.html

```diff
@@ -201,7 +201,7 @@ <h2 class="meetup-date">24 nov</h2>
     });
 </script>
 <script>
-  fetch('https://api.github.com/repos/devfsa/meetups/issues/3/comments', { headers: { "Accept": "text" } })
+  fetch('https://api.github.com/repos/devfsa/meetups/issues/4/comments', { headers: { "Accept": "text" } })
```

Comment:
This is not related with "updating title split". You should fix this in another PR.. perhaps doing some automation that avoid u to update code file every time a meetup is created

Reply:
The endpoint https://api.github.com/repos/devfsa/meetups/issues return all open issues. It can be used to auto update the next meetup id.
However, there is not enough information to fit the left side of the board.
<img width="315" alt="captura de tela 2018-11-29 as 12 36 19" src="https://user-images.githubusercontent.com/7540050/49222446-cabf8c00-f3d3-11e8-9ea0-ac6ed9b31c58.png">
To really automate I think that we should integrate with [Meetups Api](https://www.meetup.com/meetup_api/docs/:urlname/events/#list)

Reply:
If we agree that all issues of meetup editions must have title: `Edição $NUM - $DATEISO` we can do this:
```js
const meetupTitleRE = /edição \d+\s*-\s*\d{2}\/\d{2}\/\d{4}/i;
fetch('https://api.github.com/repos/devfsa/meetups/issues?state=open')
  .then(r => r.json())
  .then(issues => issues.filter(i => i.title.match(meetupTitleRE))[0])
  .then(issue => fetch(issue.comments_url).then(r => r.json()))
  .then(meetupTalks => doStuff(meetupTalks))
```

Or if we define that the standard for meetup talks issues will be some label added to the issue:
```js
fetch('https://api.github.com/repos/devfsa/meetups/issues?state=open')
  .then(r => r.json())
  .then(issues => issues.filter(i => i.labels.some(label => label.name === 'meetupTalks'))[0])
  .then(issue => fetch(issue.comments_url).then(r => r.json()))
  .then(meetupTalks => doStuff(meetupTalks))
```